specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -689,38 +720,79 @@
         }
       }
     },
+    "Pem": {
+      "type": "string"
+    },
+    "PemArray": {
+      "items": {
+        "$ref": "#/definitions/Pem"
+      },
+      "type": "array"
+    },
     "ReceiptContents": {
```

Comment:
this type seems having lots of breaking changes?

Reply:
@lynshi Can you speak to why we need this added? We re updating based on this https://github.com/microsoft/CCF/blob/af9c42cc3077724e48c4ec8ee50386cbc2308af4/doc/schemas/node_openapi.json#L545